Pin definition changed for ANET Board:

ralf-e opened the pull request:
LED D1 on ANET 3D Board is connected to PA4 (Arduino Pin27).
Anet 3D Board Schematic in my repository "ANET-3D-Board-V1.0"

changed: LED_BUILTIN = 27 (PA4)

added Pin-Description for ATMEGA1284 TQFP/QFM/MLF (44pin SMT)

Commit:
Pin definition changed for ANET Board:
LED 1 on ANET 3D Board: LED_BUILTIN = 27 (PA4)

added Pin-Description for ATMEGA1284 TQFP/QFM/MLF (44pin SMT)

oderwat commented:
OK. But the only change to actual code I see is `#define EXTERNAL_NUM_INTERRUPTS 3` right? So this is mostly about the documentation inside the comments?

ralf-e committed:
Update pins_arduino.h

Anet 3D Board: LED D1 connected to ATmrega1284P PA4 (Arduino Pin 27)
LED_BUILTIN = 27;

ralf-e commented:
Sorry, wrong version of pins_arduino.h. Now changed....

oderwat commented:
OK. But why two different pin layouts? The Anet board always has the same chip doesn't it?

ralf-e commented:
on the ANET Board ther is only the 44pin SMT version.  The 40-pin layout is the original sanguinololu Layout from your repository. On the sanguilolou the Pin-definition (PA0..PA7 > Arduino Pins 31..24) is diffrent to other Boards  on 44-pin Atmega Arduinos PA0..PA7 > 24..31 or an "Arduino UNO Setting" PA0..PA7 > 21..14.

ralf-e commented:
I have no idea if this setting for the ANET board comes from you and Skynet was then adapted to it or somewhere else comes...
Any other pin-setting would work, but then the PIN definition in SkyNet is no longer true. 
Therefore, I have not deleted the original 40-pin description for reference

abelred commented:
Hi, I'm not a professional programmer, as I'm just trying to recover my Anet A4 delta that left blank screen after doing a firmware download, the problem is that I'm trying to compile in arduino the files that you left to download and upload to A4 delta firmware but they do not compile they give error, what can I do, thanks greetings. abelred7@gmail.com
